recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "buildbot-grid-view" %}
-{% set version = "2.3.1" %}
+{% set version = "2.4.0" %}
```

Comment:
Because everything is quotes below, including the checksum, the bot will have a hard time updating this recipe.

Reply:
I can fix the recipe once I'm back from vacation (next week).